packages/browser/package.json

```diff
@@ -39,7 +39,7 @@
     "prepublishOnly": "pnpm build"
   },
   "peerDependencies": {
-    "vitest": ">=0.34.0"
+    "vitest": "^1.0.0 || ^1.0.0-beta"
```

Comment:
I believe `^1.0.0-0` should cover both

Reply:
Great tip - changed all versions to use this now